### PR TITLE
add invocation to README

### PR DESCRIPTION
**Pull request by pongad**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,22 @@ Some areas we hope to improve over the old generator are:
...
+If you are using Go 1.11 and see error `cannot find main module`, see this [FAQ page](https://github.com/golang/go/wiki/Modules#why-does-installing-a-tool-via-go-get-fail-with-error-cannot-find-main-module).
+
+The generator works as a `protoc` plugin, get `protoc` from [google/protobuf](https://github.com/protocolbuffers/protobuf).
+
+Invocation
+----------
+`protoc --go_gapic_out [OUTPUT_DIR] --go_gapic_opt 'package/path/url;name' a.proto b.proto`
+
+The `go_gapic_opt` flag is necessary because we need to know where to generated file will live.
+The substring before the semicolon is the import path of the package eg `github.com/username/awesomeness`.
```
Comment (jba) [inline README.md]: "package, e.g."

File: README.md [hunk 2]
```diff
@@ -16,6 +16,22 @@ Some areas we hope to improve over the old generator are:
...
+The substring after the semicolon is the name of the package used in the `package` statement.
+Idiomatically the name is last element of the path but it needs not be, eg if the last element of the path is a version.
```
Comment (jba) [inline README.md]: "...but it need not be. For instance, the last element of the path might be..."

File: README.md (hunk 1 above)
Comment (pongad) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (pongad) [reply README.md]: Done

**Comment by pongad** [comment]
@jba PTAL

File: README.md [hunk 3]
```diff
@@ -16,6 +16,24 @@ Some areas we hope to improve over the old generator are:
...
+The generator works as a `protoc` plugin, get `protoc` from [google/protobuf](https://github.com/protocolbuffers/protobuf).
+
+Invocation
+----------
+`protoc --go_gapic_out [OUTPUT_DIR] --go_gapic_opt 'package/path/url;name' a.proto b.proto`
+
+The `go_gapic_opt` flag is necessary because we need to know where to generated file will live.
+The substring before the semicolon is the import path of the package, e.g. `github.com/username/awesomeness`.
+The substring after the semicolon is the name of the package used in the `package` statement.
+Idiomatically the name is last element of the path but it needs not be.
```
Comment (jba) [inline README.md]: s/needs/need/

**Pull request merged**